panel_database/migrations/0005_auto_20231017_1221.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.4 on 2023-10-17 12:21
```

Comment:
did you mean to commit a migration?

Reply:
yes, i already have some migrations in the repo.
That way i can just migrate after pulling the code